metadata/src/lib.rs

```diff
@@ -190,11 +198,40 @@ impl Metadata for Album {
     }
 }
 
+impl Metadata for Playlist {
+    type Message = protocol::playlist4changes::SelectedListContent;
+
+    fn request_url(id: SpotifyId) -> String {
+        format!("hm://playlist/v2/playlist/{}", id.to_base62())
+    }
+
+    fn parse(msg: &Self::Message, _: &Session) -> Self {
+
+        let tracks = msg
+            .get_contents()
+            .get_items()
+            .iter()
+            .map(|item| {
+                let uri_split = item.get_uri().split(":");
+                let uri_parts: Vec<&str> = uri_split.collect();
+                SpotifyId::from_base62(uri_parts[2]).unwrap()
```

Comment:
I think we could do with a `SpotifyId::from_uri` helper that does the split and grabs the correct chunk. Maybe a different PR for that.

Reply:
Good suggestion!